web/source/keymanweb.js

```diff
@@ -3525,22 +3535,106 @@
+            var mutation = mutations[i];
+            
+            for(var j=0; j < mutation.addedNodes.length; j++) {
+              var addedNode = mutation.addedNodes[j];
+              var lcTagName = "";
+              
+              if(addedNode.tagName) {
+                lcTagName = addedNode.tagName.toLowerCase();
+              }
+              
+              var childAdditions = [];
+              var arrayFromNodeList = function(nl) {
+                var res = [];
+                for(var i=0; i < nl.length; i++) {
+                  res.push(nl[i]);
+                }
+                return res;
+              }
+
+              // Will need to handle this in case of child elements in a newly-added element with child elements.
+              if(addedNode.getElementsByTagName) {
+                childAdditions = childAdditions.concat(
+                  arrayFromNodeList(addedNode.getElementsByTagName('input')),
+                  arrayFromNodeList(addedNode.getElementsByTagName('textarea')),
+                  arrayFromNodeList(addedNode.getElementsByTagName('iframe'))
+                );
+              }
+
+              if(lcTagName == 'input' || lcTagName == 'textarea' || lcTagName == 'iframe') {
+                dirtyFlag = true;
+                keymanweb._MutationAdditionObserved(addedNode);
+              }
+
+              for(k = 0; k < childAdditions.length; k++) {
+                dirtyFlag = true;
+                keymanweb._MutationAdditionObserved(childAdditions[k]);
+              }
+            }
+
+            // There also exists a 'mutation.removedNodes' array.  We'll need to address that for issue #63.
+
+            // After all mutations have been handled, we need to recompile our .sortedInputs array.
+            if(dirtyFlag) {
+              keymanweb.listInputs();
+            }
+          }
+        });
+
+      var observationConfig = { childList: true, subtree: true };
+      keymanweb.mutationObserver.observe(observationTarget, observationConfig);
+
     // Set exposed initialization flag to 2 to indicate deferred initialization also complete
     keymanweb['initialized']=2;
-  }  
+  }
+
+  // Used by the MutationObserver event handler to properly setup any elements dynamically added to the document post-initialization.
+  keymanweb._MutationAdditionObserved = function(Pelem) {
+    if(!device.touchable) {
+      // keymanweb.attachToControl is written to handle iframes, but setupDesktopElement is not.
+      if(keymanweb.setupDesktopElement(Pelem)) {
+        keymanweb.attachToControl(Pelem);
+      } else if(Pelem.tagName.toLowerCase() == 'iframe') {
+        //Problem:  the iframe is loaded asynchronously, and we must wait for it to load fully before hooking in.
+
+        var attachFunctor = function() {  // Triggers at the same time as iframe's onload property, after its internal document loads.
+          keymanweb.attachToControl(Pelem);
+        }
```

Comment:
You are declaring a variable and assigning a function to it. While Javascript does let you be lazy about `;`, I prefer to include them always to avoid issues such as https://stackoverflow.com/a/1169596/1836776. So, e.g. if `attachFunctor` had a value assigned to it, we'd tend to add a semicolon just coz, viz:
````
var attachFunctor = 3;
````

Therefore:
````
var attachFunctor = function() {  // Triggers at the same time as iframe's onload property, after its internal document loads.
  keymanweb.attachToControl(Pelem);
};
````